content/applications/hr/recruitment/new_job.rst

```diff
@@ -143,68 +149,241 @@ Sections are indicated by a gray background, while questions have a white backgr
+^^^^^^
+
+If the question should only appear if a previous question is answered in a specific way, activate
+the :guilabel:`Conditional Display` option. A new :guilabel:`Triggering Question` field appears.
+Using the drop-down menu, select the previous question this new question is linked to. Once a
+previous question is selected, a :guilabel:`Triggering Answer` field appears. Select the answer that
+must be selected in order for the new question to appear.
+
+The next option is :guilabel:`Show Comments Field`. Activate the option if there should be a comment
+displayed to the applicant. If activated, a :guilabel:`Comment Message` field appears. Enter the
+text to appear in the :guilabel:`Comment Message` field.
+
+.. example::
+   To further illustrate a triggering question, the following is an example that is applicable to
+   recruitment. The question, `Do you have management experience?` is already added. A new question
+   is then added, `How many years of experience?`. This question should *only* appear if the
+   applicant selected `Yes` to the question `Do you have management experience?`.
+
+   To properly configure this example, activate the :guilabel:`Conditional Display` option. Then,
+   select `Do you have management experience?` as the :guilabel:`Triggering Question`. Then, select
+   :guilabel:`Yes` for the triggering answer.
+
+
+   .. image:: new_job/layout.png
+      :align: center
+      :alt: The layout field when properly configured for a conditional question.
+
+Live Sessions
+^^^^^^^^^^^^^
+
+If the question is for a live session, activate the :guilabel:`Question Time Limit` option. A
+:guilabel:`seconds` field appears. Enter the amount of time allotted for the applicant to enter the
+answer, in seconds.
+
+Options
+~~~~~~~
+
+Next, configure the various options for the interview form. Click the :guilabel:`Options` tab to
+view all the options to configure, by category.
 
 Questions
-~~~~~~~~~
+*********
 
-- :guilabel:`Layout`: Choose how the questions should be displayed.
+- :guilabel:`Pagination`: Choose how the questions should be displayed.
 
   - :guilabel:`One page with all the questions`: Display all sections and questions at the same
     time.
   - :guilabel:`One page per section`: Display each section with the corresponding questions on an
     individual page.
   - :guilabel:`One page per question`: Display a single question on each page.
 
-- :guilabel:`Progression Mode`: Display the progress the candidate is making, either as a
-  :guilabel:`Percentage`, or as a :guilabel:`Number`.
+
+- :guilabel:`Question Selection`: Choose which questions are presented.
+
+  - :guilabel:`All questions`: Display the entire form, with all questions form all sections.
+  - :guilabel:`Randomized per Section`: Display only a random selection of questions from each
+    section.
+
+.. note::
+   Although it is a selectable option, the :guilabel:`Randomized per section` option is not
+   typically selected for an interview form.
```

Comment:
I would change this (which veers a bit too close to advice on hiring and not advice on using the product) and explain why it isn't usually used. Mainly that it should only be used if they are ok with receiving partial information/an incomplete survey from every applicant.

Reply:
Good point- I guess I said this because surveys are used for a lot of things, and only when making an interview form I would think you didn't want to do this. Good suggestion!